<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenMenu</class>
 <widget class="QDialog" name="OpenMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/monokuma-green.png</normaloff>:/monokuma-green.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="treeFileList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="indentation">
      <number>15</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Chapter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRefresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons.qrc">
           <normaloff>:/arrow_refresh.png</normaloff>:/arrow_refresh.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblChapter">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Meiryo UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblScene">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Meiryo UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Room</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblRoom">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Meiryo UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblMode">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Meiryo UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearch"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeFileList</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>OpenMenu</receiver>
   <slot>changeSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeFileList</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>OpenMenu</receiver>
   <slot>doubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSearch</sender>
   <signal>textEdited(QString)</signal>
   <receiver>OpenMenu</receiver>
   <slot>findDirectory(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>OpenMenu</receiver>
   <slot>refresh_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>doubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>findDirectory(QString)</slot>
  <slot>refresh_list()</slot>
 </slots>
</ui>
